core/module.h

```diff
@@ -182,9 +171,9 @@ class Module {
         ogates() {}
   virtual ~Module() {}
 
-  pb_error_t Init(const google::protobuf::Any &arg);
+  pb_error_t RunInit(const google::protobuf::Any &arg);
```

Comment:
I think this one should be moved to the private section, since:
- A derived class does not need to override this function, but the comment above says something different
- If bessctl.cc is the only user (besides module_test.cc), it should be friended.
The name RunInit is too generic and it does not say anything about the difference between RunInit and Init. How about InitWithGenericArg or something like that?